dbxio/delta/table_commands.py

```diff
@@ -225,12 +226,14 @@ def bulk_write_table(
     pa_table = create_pa_table(columnar_table, schema=dbxio_table.schema)
     pa_table_as_bytes = pa_table2parquet(pa_table)
 
-    abs_client = get_blob_servie_client(
-        abs_name,
-        az_cred_provider=client.credential_provider.az_cred_provider,
-    ).get_container_client(abs_container_name)
+    object_storage = ObjectStorageClient.from_storage_options(
+        scheme='abfss',  # TODO: make cloud-agnostic
```

Comment:
Can we move this to `Settings()`? To setup schemes at least with env vars.

Reply:
Moved it to the `Settings`